template/lambda-sqs-worker/serverless.yml

```diff
@@ -4,21 +4,32 @@ configValidationMode: error
 
 params:
   default:
+    datadogApiKeySecretArn: 'TODO: arn:aws:secretsmanager:${aws:region}:${aws:accountId}:secret:SECRET-NAME'
     description: <%- description %>
   dev:
-    deploymentBucket: 'TODO: deploymentBucketName'
+    deploymentBucket: 'TODO: deployment-bucket-name'
     isProduction: false
   prod:
-    deploymentBucket: 'TODO: deploymentBucketName'
+    deploymentBucket: 'TODO: deployment-bucket-name'
     isProduction: true
 
 custom:
+  datadog:
+    apiKeySecretArn: ${param:datadogApiKeySecretArn}
+    enableDDLogs: false
+    # TODO: enable Datadog extension
+    enabled: false
```

Comment:
Would there be any reason why they wouldn't enable it? Don't they need the extension to be able to get traces etc?

Reply:
The intent is for the template to be largely self-contained and deployable without having to line up all every production concern beforehand so it's an easier jumping-off point. I imagine internal team templates can be stricter and the concern will likely be alleviated once our internal developer platform is further along.

Another example of this is how the stack comes with its own KMS key when that wouldn't be our recommendation in production (and we should probably clarify that). This is also analogous to https://github.com/seek-oss/skuba/blob/0a3316164148e013b758c4125d1d9b7a31ba5821/template/koa-rest-api/.gantry/common.yml#L6-L8